build.gradle

```diff
@@ -36,4 +38,28 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
+apiValidation {
+    ignoredPackages += [
+            'io/customer/base/extenstions',
+            'io/customer/sdk/api',
+            'io/customer/sdk/data/store',
+            'io/customer/sdk/queue',
+            'io/customer/sdk/extensions',
+            'io/customer/sdk/util',
```

Comment:
Along with my comment below on `ignoredProjects`, I believe we can ignore `base` module completely. 

Mind adding some comments to this file indicating why a package should or should not be ignored? From this list, I am not sure why these were chosen?